yt/utilities/lib/misc_utilities.pyx

```diff
@@ -568,7 +568,8 @@ def get_box_grids_below_level(
 @cython.cdivision(True)
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def obtain_position_vector(data):
+def obtain_position_vector(
+    data, field_names = ('x', 'y', 'z')):
```

Comment:
should maybe add a note that this may result in an ambiguous access

Reply:
I agree with this.